public/app/features/plugins/admin/types.ts

```diff
@@ -255,6 +255,7 @@ export enum PluginTabLabels {
   USAGE = 'Usage',
   IAM = 'IAM',
   CHANGELOG = 'Changelog',
+  RIGHTPANEL = 'Plugin details',
```

Comment:
Why not just call this `PLUGINDETAILS`? I feel that `RIGHTPANEL` makes sense now but in a year when you don't have this PR fresh in mind the naming is a bit confusing.

Reply:
+1